src/components/common/AgreementType.vue

```diff
@@ -21,43 +21,45 @@
       </div>
     </template>
 
-    <!-- EDIT SECTION -->
-    <div v-if="isTypeBcUlcCompany || isTypeBcCcc" :class="{ 'invalid-section': showErrorSummary && getShowErrors }">
-      <v-checkbox
-        v-for="(item, index) in getIncorporationAgreementDocuments"
-        v-model="agreementType"
-        class="ml-6 mt-0 agreement-option-list"
-        :id="`agreement-type-${item.code}`"
-        :key="index"
-        :value="item.code"
-        @change="changeAgreementType()"
-      >
-        <template v-slot:label>
-          <div v-html="item.description" class="ml-6 py-4 agreement-option" />
-        </template>
-      </v-checkbox>
-    </div>
-
-    <!-- EDIT SECTION -->
-    <div class="py-8 px-6" v-else :class="{ 'invalid-section': showErrorSummary && getShowErrors }">
-      <v-radio-group
-        class="mt-0 pt-0"
-        hide-details
-        v-model="agreementType"
-        @change="changeAgreementType()"
-      >
-        <v-radio
+    <template v-if="!isSummary">
+      <!-- EDIT SECTION -->
+      <div v-if="isTypeBcUlcCompany || isTypeBcCcc" :class="{ 'invalid-section': showErrorSummary && getShowErrors }">
```

Comment:
Specifically, this section is for entity types that have a single incorporation agreement document, right? It would be make the code more generic if you checked that instead. Thoughts?

Reply:
Sure can be refactored.